target/riscv/cheri-archspecific-early.h

```diff
@@ -84,6 +84,16 @@ enum CheriSCR {
     CheriSCR_MTDC = 29,
     CheriSCR_MScratchC = 30,
     CheriSCR_MEPCC = 31,
+
+    /*
+     * TODO(am2419): Register indices are placeholders,
+     * awaiting full specification of background registers.
+     */
+    CheriSCR_BSTCC = 32,
+    CheriSCR_BSTDC = 33,
+    CheriSCR_BSScratchC = 34,
+    CheriSCR_BSEPCC = 35,
+    CheriSCR_MAX,
```

Comment:
Can we have more than 32? I'm not sure what the instruction format is.

Reply:
I assume not, which is why I commented it with the TODO, specifiying that these are just placeholders. We currently don't have the instruction to access them implemented.